Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBarButtonsView.swift

```diff
@@ -121,6 +122,32 @@ final class InputBarButtonsView: UIView {
         currentRow = rowIndex
         buttonRowTopInset.constant = CGFloat(rowIndex) * constants.buttonsBarHeight
         UIView.animate(easing: .easeInOutExpo, duration: animated ? 0.35 : 0, animations: layoutIfNeeded)
+        setupAccessibility()
+    }
+
+    private func setupAccessibility() {
+        if multilineLayout {
+            let firstRowButtons = [UIButton](buttons.prefix(customButtonCount))
+            let secondRowButtons = [UIButton](buttons.suffix(buttons.count - customButtonCount))
+            
+            buttons.forEach {
+                var isAccessibilityElement = false
+                if currentRow == 0 {
+                    isAccessibilityElement = firstRowButtons.contains($0)
+                } else {
+                    isAccessibilityElement = secondRowButtons.contains($0)
+                }
+                $0.isAccessibilityElement = isAccessibilityElement
+
+            }
```

Comment:
```suggestion
            buttons.forEach {
                $0.isAccessibilityElement = currentRow == 0 
                   ? firstRowButtons.contains($0)
                   : secondRowButtons.contains($0)
            }
```
just suggesting another way, feel free to choose what you prefer :)